mlton/ssa/ssa-tree.sig

```diff
@@ -252,6 +264,9 @@ signature SSA_TREE =
              * for block b to yield v', then visiting b's descendents,
              * then applying v' ().
              *)
+	    val functionSize: int * int option -> (Exp.t -> int) * (Transfer.t -> int) -> t -> int * bool
+	    val default: (Exp.t -> int) * (Transfer.t -> int)
+	    val functionGT: int option -> t -> bool
```

Comment:
The newly added functions to the signature have separated the comment about the `dfs` function from the specification of the `dfs` function.